harvard-stellenbosch-university.csl

```diff
@@ -63,7 +63,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="10" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
+      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="" et-al-use-first="" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
```

Comment:
`et-al-min="" et-al-use-first=""`

You need to delete these attributes, not just assign them an empty value.